mxcube3/routes/Login.py

```diff
@@ -42,21 +44,36 @@ def login():
 
     try:
         login_res = limsutils.lims_login(loginID, password)
-        inhouse = limsutils.lims_is_inhouse(login_res)
 
+        if login_res['status']["code"] is not "ok":
```

Comment:
I know very picky and not so important  but the usage of " and ' is a bit inconsistent

Reply:
you have good eyes

Reply:
While we are nitpicking:
'!= "ok"' would be better, 'is not' dep[enmds on how strings are stored internally, whether both cases point to the same string object,. or different but identical objects.

Reply:
very true, and maybe more important ...